opentelemetry-sdk/src/metrics/mod.rs

```diff
@@ -199,6 +200,20 @@ mod tests {
         counter_aggregation_helper(Temporality::Delta);
     }
 
+    #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
+    async fn histogram_aggregation_cumulative() {
+        // Run this test with stdout enabled to see output.
+        // cargo test histogram_aggregation_cumulative --features=metrics,testing -- --nocapture
```

Comment:
nit - this comment is valid for all the tests in general, do we need to add specifically for these tests ?

Reply:
No strong opinion. I think it makes it easy to copy paste locally, to run one test alone. (I personally use it heavily)